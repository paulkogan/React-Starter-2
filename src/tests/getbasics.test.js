import React, {} from "react";
//import {BrowserRouter, Route, Switch, Link, NavLink } from "react-router-dom";
import DealDetailsPage, {} from "../components/deal-details-page"
import renderer from 'react-test-renderer';


// use: npm test -- -u`
it('renders Deal Page correctly', () => {
  const tree = renderer
    .create(<DealDetailsPage nid={20}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});




const add = (a,b) => a+b;
test("should add two numbers", () => {

          const result = add(3,4);
          expect(result).toBe(7);


});
