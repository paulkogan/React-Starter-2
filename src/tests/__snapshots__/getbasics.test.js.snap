// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Deal Page correctly 1`] = `
<div>
  <div
    className="div-border"
  >
    <u>
       Deal Financials for  
       
    </u>
    <br />
    Entity_ID: 
    20
      
    <br />
    Deal_ID: 
    1001
      
    <br />
    total_assets: 
    $0.00
      
    <br />
    total_debt: 
    $0.00
      
    <br />
    equity_value: 
    $0.00
      
    <br />
    notes: 
      
    <br />
    <br />
  </div>
  <br />
  <br />
  <br />
</div>
`;
